README.md

```diff
@@ -48,6 +48,12 @@ media_storage_providers:
     # to S3. Each thread manages a single connection. Default is 40.
     #
     #threadpool_size: 20
+
+    # Set request_checksum_calculation or response_checksum_validation
```

Comment:
Please add some words about *why* a sysadmin may want to set these options (i.e. link to AWS' docs that explain what "when_supported" and "when_required" mean).

Reply:
Will do!